.eslintrc

```diff
@@ -53,6 +53,7 @@
         "plugin:jest/all"
       ],
       "rules": {
+        "jest/expect-expect": "off",
```

Comment:
Before, there was a linting error when `e2e` tests didn't have an `expect()` call.

But `expect()` isn't really needed, as the `find*` functions do almost the same thing as it.